<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="wiΩdth=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Timothy Tan | publications</title>
    <meta name="author" content="Timothy  Tan" />
    <meta name="description" content="preprints, articles in press, and publications in reverse chronological order. * and ^ denote equal contribution." />
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website" />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light" />

    <!-- Styles -->
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://timothyjtan.github.io/publications/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark" />

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="https://timothyjtan.github.io/">Timothy Tan</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a>
              </li>

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">publications</h1>
            <p class="post-description">preprints, articles in press, and publications in reverse chronological order. * and ^ denote equal contribution.</p>
          </header>

          <article>
            <p>Preprints and publications are also available on <a href="https://scholar.google.com/citations?user=doEQkUUAAAAJ" target="_blank" rel="noopener noreferrer">Google Scholar</a>.</p>
<!-- _pages/publications.md -->
<div class="publications">

<h2 class="year"><b>Preprints</b></h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Ouyang2022Probing" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">Probing the biophysical constraints of SARS-CoV-2 spike N-terminal domain using deep mutational scanning</div>
          <!-- Author -->
          <div class="author">Ouyang WO*, <em>Tan TJC*</em>, Lei R, Song G, Kieffer C, Andrabi R, Matreyek KA, and Wu NC
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> (2022)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.biorxiv.org/content/10.1101/2022.06.20.496903v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Ouyang2022Probing.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
            <a href="/assets/pdf/Ouyang2022Probing_supp.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Increasing the expression level of the SARS-CoV-2 spike (S) protein has been critical for COVID-19 vaccine development. While previous efforts largely focused on engineering the receptor-binding domain (RBD) and the S2 subunit, the N-terminal domain (NTD) has been long overlooked due to the limited understanding of its biophysical constraints. In this study, the effects of thousands of NTD single mutations on S protein expression were quantified by deep mutational scanning. Our results revealed that in terms of S protein expression, the mutational tolerability of NTD residues was inversely correlated with their proximity to the RBD and S2. We also identified NTD mutations at the interdomain interface that increased S protein expression without altering its antigenicity. Overall, this study not only advances the understanding of the biophysical constraints of the NTD, but also provides invaluable insights into S-based immunogen design.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Liu2022Evolutionary" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">The evolutionary potential of the influenza A virus hemagglutinin is highly constrained by intersegment epistasis</div>
          <!-- Author -->
          <div class="author">Liu T, Wang Y,
                  <em>Tan TJC</em>, Wu NC, and Brooke CB
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> (2022)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.biorxiv.org/content/10.1101/2022.05.19.492711v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Liu2022Evolutionary.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The ongoing antigenic evolution of the influenza A virus (IAV) hemagglutinin (HA) gene limits efforts to effectively control the spread of the virus in the human population through vaccination. The factors that influence and constrain the evolutionary potential of the HA gene remain poorly understood. Efforts to understand the mechanisms that govern HA antigenic evolution typically examine the HA gene in isolation and ignore the importance of balancing HA receptor-binding activities with the receptor-destroying activities of the viral neuraminidase (NA) for maintaining viral fitness. We hypothesized that the need to maintain functional balance with NA significantly constrains the evolutionary potential of the HA gene. We used deep mutational scanning to show that variation in NA activity significantly reshapes the HA fitness landscape by modulating the overall mutational robustness of the HA protein. Consistent with this, we observe that different NA backgrounds support the emergence of distinct repertoires of HA escape variants under neutralizing antibody pressure. Our results reveal a critical role for intersegment epistatic interactions in shaping the evolutionary potential of the HA gene.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Lei2022Prevalence" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">Prevalence and mechanisms of evolutionary contingency in human influenza H3N2 neuraminidase</div>
          <!-- Author -->
          <div class="author">Lei R, 
                  <em>Tan TJC</em>, Hernandez Garcia A, Wang Y, Teo C, Gopan G, Dargani ZT, Teo QW, Graham CS, Nair SK, and Wu NC
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> (2022)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.biorxiv.org/content/10.1101/2022.02.24.481718v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Lei2022Prevalence.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
            <a href="/assets/pdf/Lei2022Prevalence_supp.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Neuraminidase (NA) of human influenza H3N2 virus has evolved rapidly and been accumulating mutations for more than half-century. However, biophysical constraints that govern the evolutionary trajectories of NA remain largely elusive. Here, we show that among 70 natural mutations that are present in the NA of a recent human H3N2 strain, &gt;10% are deleterious for an ancestral strain. By mapping the permissive mutations using combinatorial mutagenesis and next-generation sequencing, an extensive epistatic network is revealed. Biophysical and structural analyses further demonstrate that certain epistatic interactions can be explained by non-additive stability effect, which in turn modulates membrane trafficking and enzymatic activity of NA. Additionally, our results suggest that other biophysical mechanisms also contribute to epistasis in NA evolution. Overall, these findings not only provide mechanistic insights into the evolution of human influenza NA and elucidate its sequence-structure-function relationship, but also have important implications for the development of next-generation influenza vaccines.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Liang2022Egg" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">Egg-adaptation pathway of human influenza H3N2 virus is contingent on natural evolution</div>
          <!-- Author -->
          <div class="author">Liang W, 
                  <em>Tan TJC</em>, Wang Y, Lv H, Sun Y, Bruzzone R, Mok CKP, and Wu NC
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> (2022)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.biorxiv.org/content/10.1101/2022.01.31.478576v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Liang2022Egg.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
            <a href="/assets/pdf/Liang2022Egg_supp.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Egg-adaptive mutations in influenza hemagglutinin (HA) often emerge during the production of egg-based seasonal influenza vaccines, which contribute to the largest share in the global influenza vaccine market. While some egg-adaptive mutations have minimal impact on the HA antigenicity (e.g. G186V), others can alter it (e.g. L194P). Here, we show that the preference of egg-adaptation pathway in human H3N2 HA is strain-dependent. In particular, Thr160 and Asn190, which are found in many recent H3N2 strains, restrict the emergence of L194P but not G186V. Our results further suggest that natural amino acid variants at other HA residues also play a role in determining the egg-adaptation pathway. Consistently, recent human H3N2 strains from different clades acquire different mutations during egg passaging. Overall, these results demonstrate that natural mutations in human H3N2 HA can influence the egg-adaption pathway, which has important implications in seed strain selection for egg-based influenza vaccine.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Schulte2021Caprin" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">Caprin-1 binding to the critical stress granule protein G3BP1 is regulated by pH</div>
          <!-- Author -->
          <div class="author">Schulte T, Panas MD, Williams L, Kedersha N, Fleck JS, 
                  <em>Tan TJC</em>, Olsson A, Morro AM, Hanke L, Nilvebrant J, Giang KA, Nygren P-Å, Anderson P, Achour A, and McInerney GM
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> (2021)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.biorxiv.org/content/10.1101/2021.02.05.429362v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Schulte2021Caprin.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>G3BP is the central hub within the protein-RNA interaction network of stress-induced bio-molecular condensates known as stress granules (SG). The SG-associated proteins Caprin-1 and USP10 exhibit mutually exclusive binding to the structured NTF2-domain of G3BP1, thereby regulating G3BP1-mediated condensation, but with opposite effects: Caprin-1 promotes but USP10 inhibits SG formation. Herein, we present the crystal structure of G3BP1-NTF2 in complex with a Caprin-1 derived short linear motif (SLiM), which provides a molecular understanding for the mutually exclusive binding of USP10 and Caprin-1 to G3BP1. Caprin-1 but not USP10 contacts two G3BP1-NTF2 histidine residues, which was confirmed using biochemical, biophysical and cellular biological binding assays. G3BP1/Caprin-1 interactions disrupted via point mutations resulted in fewer and smaller SG condensates. In addition, biochemical binding assays demonstrated reduced binding of Caprin-1 to G3BP1 at lower pH values. Finally, ratiometric pH sensitive measurements of SGs revealed a substantial drop in pH compared to the adjacent cytosol, suggesting that reduced pH can fine-tune and regulate the G3BP1-mediated interaction network via a NTF2-mediated pH-sensitive SLiM-selection mechanism.</p>
          </div>
        </div>
      </div>
</li>
</ol>

<h2 class="year"><b>In Press</b></h2>
  <ol class="bibliography"></ol>

<h2 class="year"><b>2022</b></h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Diefenbacher2022Interactions" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">
Interactions between influenza A virus nucleoprotein and gene segment UTRs facilitate selective modulation of viral gene expression</div>
          <!-- Author -->
          <div class="author">Diefenbacher M, 
                  <em>Tan TJC</em>, Bauer DLV, Stadtmueller B, Wu NC, and Brooke C
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Journal of Virology</em>
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://journals.asm.org/doi/abs/10.1128/jvi.00205-22?af=R" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Diefenbacher2022Interactions.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The influenza A virus (IAV) genome is divided into eight negative-sense, single-stranded RNA segments. Each segment exhibits a unique level and temporal pattern of expression, however the exact mechanisms underlying the patterns of individual gene segment expression are poorly understood. We previously demonstrated that a single substitution in the viral nucleoprotein (NP:F346S) selectively modulates neuraminidase (NA) gene segment expression while leaving other segments largely unaffected. Given what is currently known about NP function, there is no obvious explanation for how changes in NP can selectively modulate the replication of individual gene segments. We found that the specificity of this effect for the NA segment is virus strain specific and depends on the UTR sequences of the NA segment. While the NP:F346S substitution did not significantly alter the RNA binding or oligomerization activities of NP <em>in vitro</em>, it specifically decreased the ability of NP to promote NA segment vRNA synthesis. In addition to NP residue F346, we identified two other adjacent aromatic residues in NP (Y385 &amp; F479) capable of similarly regulating NA gene segment expression, suggesting a larger role for this domain in gene-segment specific regulation. Our findings reveal a new role for NP in selective regulation of viral gene segment replication and demonstrate how the expression patterns of individual viral gene segments can be modulated during adaptation to new host environments.</p>
          </div>
        </div>
      </div>
</li>
</ol>

  

  <h2 class="year"><b>2021</b></h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Reuper2021Arabidopsis" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">
<em>Arabidopsis thaliana</em> G3BP ortholog rescues mammalian stress granule phenotype across kingdoms</div>
          <!-- Author -->
          <div class="author">Reuper H, Götte B, Williams L, 
                  <em>Tan TJC</em>, McInerney GM, Panas MD^, and Krenz B^
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>International Journal of Molecular Sciences</em> (2021)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.mdpi.com/1422-0067/22/12/6287" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Reuper2021Arabidopsis.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
            <a href="/assets/pdf/Reuper2021Arabidopsis_supp.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Stress granules (SGs) are dynamic RNA–protein complexes localized in the cytoplasm that rapidly form under stress conditions and disperse when normal conditions are restored. The formation of SGs depends on the Ras-GAP SH3 domain-binding protein (G3BP). Formations, interactions and functions of plant and human SGs are strikingly similar, suggesting a conserved mechanism. However, functional analyses of plant G3BPs are missing. Thus, members of the <em>Arabidopsis thaliana</em> G3BP (AtG3BP) protein family were investigated in a complementation assay in a human G3BP knock-out cell line. It was shown that two out of seven AtG3BPs were able to complement the function of their human homolog. GFP-AtG3BP fusion proteins co-localized with human SG marker proteins Caprin-1 and eIF4G1 and restored SG formation in G3BP double KO cells. Interaction between AtG3BP-1 and -7 and known human G3BP interaction partners such as Caprin-1 and USP10 was also demonstrated by co-immunoprecipitation. In addition, an RG/RGG domain exchange from Arabidopsis G3BP into the human G3BP background showed the ability for complementation. In summary, our results support a conserved mechanism of SG function over the kingdoms, which will help to further elucidate the biological function of the Arabidopsis G3BP protein family.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Tan2021Sequence" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">Sequence signatures of two IGHV3-53/3-66 public clonotypes to SARS-CoV-2 receptor binding domain</div>
          <!-- Author -->
          <div class="author">
                  <em>Tan TJC*</em>, Yuan M*, Kuzelka K, Padron GC, Beal JR, Chen X, Wang Y, Rivera-Cardona J, Zhu X, Stadtmueller BM, Brooke CB, Wilson IA^, and Wu NC^
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Nature Communications</em> (2021)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.nature.com/articles/s41467-021-24123-7" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Tan2021Sequence.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
            <a href="/assets/pdf/Tan2021Sequence_supp.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Since the COVID-19 pandemic onset, the antibody response to SARS-CoV-2 has been extensively characterized. Antibodies to the receptor binding domain (RBD) on the spike protein are frequently encoded by IGHV3-53/3-66 with a short complementarity-determining region (CDR) H3. Germline-encoded sequence motifs in heavy chain CDRs H1 and H2 have a major function, but whether any common motifs are present in CDR H3, which is often critical for binding specificity, is not clear. Here, we identify two public clonotypes of IGHV3-53/3-66 RBD antibodies with a 9-residue CDR H3 that pair with different light chains. Distinct sequence motifs on CDR H3 are present in the two public clonotypes that seem to be related to differential light chain pairing. Additionally, we show that Y58F is a common somatic hypermutation that results in increased binding affinity of IGHV3-53/3-66 RBD antibodies with a short CDR H3. These results advance understanding of the antibody response to SARS-CoV-2.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Ravi2021Virus" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">Virus-induced activation of the rac1 protein at the site of respiratory syncytial virus assembly is a requirement for virus particle assembly on infected cells</div>
          <!-- Author -->
          <div class="author">Ravi LI, 
                  <em>Tan TJC</em>, Tan BH, and Sugrue RJ
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Virology</em> (2021)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.sciencedirect.com/science/article/abs/pii/S0042682221000428" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The distributions of the rac1, rhoA and cdc42 proteins in respiratory syncytial virus (RSV) infected cells was examined. All three rhoGTPases were detected within inclusion bodies, and while the rhoA and rac1 proteins were associated with virus filaments, only the rac1 protein was localised throughout the virus filaments. RSV infection led to increased rac1 protein activation, and using the rac1 protein inhibitor NS23766 we provided evidence that the increased rac1 activation occurred at the site of RSV assembly and facilitated F-actin remodeling during virus morphogenesis. A non-infectious virus-like particle (VLP) assay showed that the RSV VLPs formed in lipid-raft microdomains containing the rac1 protein, together with F-actin and filamin-1 (cell proteins associated with virus filaments). This provided evidence that the virus envelope proteins are trafficked to membrane microdomains containing the rac1 protein. Collectively, these data provide evidence that the rac1 protein plays a direct role in the RSV assembly process.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        

        <!-- Entry bib key -->
        <div id="Chan2021Engineered" class="col-sm-10">
        
          <!-- Title -->
          <div class="title">An engineered decoy receptor for SARS-CoV-2 broadly binds protein S sequence variants</div>
          <!-- Author -->
          <div class="author">Chan KK, 
                  <em>Tan TJC</em>, Narayanan KK, and Procko E
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Science Advances</em> (2021)
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          
            <a href="https://www.science.org/doi/10.1126/sciadv.abf1738" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="/assets/pdf/Chan2021Engineered.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
            <a href="/assets/pdf/Chan2021Engineered_supp.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Supp</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The spike S of SARS-CoV-2 recognizes ACE2 on the host cell membrane to initiate entry. Soluble decoy receptors, in which the ACE2 ectodomain is engineered to block S with high affinity, potently neutralize infection and, because of close similarity with the natural receptor, hold out the promise of being broadly active against virus variants without opportunity for escape. Here, we directly test this hypothesis. We find that an engineered decoy receptor, sACE2_2.v2.4, tightly binds S of SARS-associated viruses from humans and bats, despite the ACE2-binding surface being a region of high diversity. Saturation mutagenesis of the receptor-binding domain followed by in vitro selection, with wild-type ACE2 and the engineered decoy competing for binding sites, failed to find S mutants that discriminate in favor of the wild-type receptor. We conclude that resistance to engineered decoys will be rare and that decoys may be active against future outbreaks of SARS-associated betacoronaviruses.</p>
          </div>
        </div>
      </div>
</li>
</ol>


</div>

          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2022 Timothy  Tan. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>

